### PR TITLE
[MRG] Add Arabic translation of tutorials/quickstart/beginner

### PR DESCRIPTION
**Pull request by mohamed-ali**
This PR pushes an Arabic version of the tutorials/quickstart/beginner notebook. At the same time, it showcases, how to use html tags to make Arabic layout display and switch correctly between RTL Arabic text and LTR code.

@lamberta Could you have a look?
Since this notebook is rather short, we can use it to see if the RTL rendering works correctly in tensorflow's website. Thanks.

### CODE REVIEW

**Comment by mohamed-ali** [comment]
@lamberta Is there a staging environment where we can see the final html output before deploying in the production website? 

Thanks.

**Comment by lamberta** [comment]
No external staging, unfortunately.

But you can play around with `nbconvert` to generate Markdown. One issue that comes to mind, wrapping Markdown with a div may not render correctly across targets. Will have to experiment. Can you file an issue on the repo tracker? Thanks

**Comment by mohamed-ali** [comment]
@lamberta You mean an issue regarding RTL support in https://github.com/tensorflow/docs-l10n/? 

Also, I think the easiest/fastest way to move forward is to do the experiment. With that in mind, I made this PR for a single tutorial to simplify doing the experiment.

**Comment by mohamed-ali** [comment]
Trying the notebook on https://nbviewer.jupyter.org/ which renders the notebook to html displays everything correctly:

https://nbviewer.jupyter.org/github/abjed/docs-l10n/blob/ar-quickstart/site/ar/tutorials/quickstart/beginner.ipynb

**Comment by mohamed-ali** [comment]
@lamberta  As you suggested, I added this issue https://github.com/tensorflow/docs-l10n/issues/129 to gather information about RTL support in tensorflow website.

**Comment by lamberta** [comment]
Hi @baheytharwat , can you please review this notebook? Once merged, we can use this to test the Arabic docs pipeline. Thanks

File: site/ar/tutorials/quickstart/beginner.ipynb [hunk 1]
```diff
@@ -0,0 +1,542 @@
...
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "BPZ68wASog_I"
+      },
+      "source": [
+        "<div dir=\"rtl\">\n",
+        "\n",
+        "قم ببناء نموذج\n",
+        "`tf.keras.Sequentia`\n",
```
Comment (baheytharwat) [inline site/ar/tutorials/quickstart/beginner.ipynb]: 'tf.keras.Sequential

Comment (baheytharwat) [inline site/ar/tutorials/quickstart/beginner.ipynb]: 'tf.keras.Sequential'

Comment (baheytharwat) [inline site/ar/tutorials/quickstart/beginner.ipynb]: 'tf.keras.Sequential'

File: site/ar/tutorials/quickstart/beginner.ipynb [hunk 2]
```diff
@@ -0,0 +1,542 @@
...
+        "colab_type": "text",
+        "id": "tgjhDQGcIniO"
+      },
+      "source": [
+        "<div dir=\"rtl\">\n",
+        "\n",
+        "تحول الدّالة \n",
+        "`tf.nn.softmax`\n",
+        " هذه النتائج ، المتمثلة في أعداد حقيقية ، إلى \"احتمالات\"\n",
+        " لكل فئة ، حيث تكون قيمة كُلّ واحدة من هذه الإحتمالات بين 0 و 1:\n",
```
Comment (baheytharwat) [inline site/ar/tutorials/quickstart/beginner.ipynb]: و يكون مجموع هذه الاحتمالات لكل القيم يُساوي 1

**Comment by mohamed-ali** [comment]
@baheytharwat Thanks for reviewing, I've just pushed a commit to add the suggested correction & rewording.

**Comment by lamberta** [comment]
Can you check "Allow edits from maintainers." on this PR. Thanks

We like to run notebooks through [nbfmt.py](https://github.com/tensorflow/docs/blob/master/tools/nbfmt.py) before merge. This normalizes the notebook JSON and will make future diffs more readable. Hope to make this part of the repo CI, but not working, yet. You're welcome to run this, too.

When reviews are finished, Reviewers please approve the pull request using the regular GitHub approval process or write "LGTM" in a comment. Then tfdocsbot will add a label that I can easily see when browsing.
With so many languages and pull requests, it's difficult for me to be involved in each one. And why we really need to build a community of translators *and* reviewers to be successful.

**Comment by mohamed-ali** [comment]
The "Allow edits from maintainers." doesn't show to me in this already created PR. I'll consider checking it in future PRs.

I'll run the nbfmt.py  script when I have sometime, maybe in an hour. However, I am editing and downloading the notebooks in my PRs from colab directly.

**Comment by mohamed-ali** [comment]
@lamberta I have just run the nbfmt.py script on the notebook. I think it's good to merge now. Let me know If anything else needs to be improved.

File: site/ar/tutorials/quickstart/beginner.ipynb [hunk 3]
```diff
@@ -0,0 +1,542 @@
...
+        "colab_type": "text",
+        "id": "DUNzJc4jTj6G"
+      },
+      "source": [
+        "<table class=\"tfo-notebook-buttons\" align=\"right\" dir=\"rtl\">\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://www.tensorflow.org/tutorials/quickstart/beginner?hl=ar\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" />إفتح المحتوى على موقع TensorFlow.org</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/docs/blob/master/site/ar/tutorials/quickstart/beginner.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />تفاعل مع المحتوى على Google Colab</a>\n",
```
Comment (lamberta) [inline site/ar/tutorials/quickstart/beginner.ipynb]:
```suggestion
        "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/docs-l10n/blob/master/site/ar/tutorials/quickstart/beginner.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />تفاعل مع المحتوى على Google Colab</a>\n",
```

Need to update the repo name. Just noticed this in the other merged PR and will send a fix

File: site/ar/tutorials/quickstart/beginner.ipynb [hunk 4]
```diff
@@ -0,0 +1,542 @@
...
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://github.com/tensorflow/docs/blob/master/site/ar/tutorials/quickstart/beginner.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />اعرض المصدر على Github</a>\n",
```
Comment (lamberta) [inline site/ar/tutorials/quickstart/beginner.ipynb]:
```suggestion
        "    <a target=\"_blank\" href=\"https://github.com/tensorflow/docs-l10n/blob/master/site/ar/tutorials/quickstart/beginner.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />اعرض المصدر على Github</a>\n",
```

File: site/ar/tutorials/quickstart/beginner.ipynb [hunk 5]
```diff
@@ -0,0 +1,542 @@
...
+        "  </td>\n",
+        "  <td>\n",
+        "    <a href=\"https://storage.googleapis.com/tensorflow_docs/docs/site/ar/tutorials/quickstart/beginner.ipynb\"><img src=\"https://www.tensorflow.org/images/download_logo_32px.png\" />نزّل الدّفتر</a>\n",
```
Comment (lamberta) [inline site/ar/tutorials/quickstart/beginner.ipynb]:
```suggestion
        "    <a href=\"https://storage.googleapis.com/tensorflow_docs/docs-l10n/site/ar/tutorials/quickstart/beginner.ipynb\"><img src=\"https://www.tensorflow.org/images/download_logo_32px.png\" />نزّل الدّفتر</a>\n",
```

**Comment by mohamed-ali** [comment]
@lamberta I committed your suggestions to link to the right repo. Thanks.
